docs/getting_started/install_ez_enterprise.md

```diff
@@ -76,18 +70,33 @@ They are available via [updates.ez.no](https://updates.ez.no).
         They will become active again if the agreement is renewed, but this process may take up to 24 hours.
         _(If the agreement is renewed before the expiry date, there will be no disruption of service.)_
 
+    ## Create project
+
+    In order to install a new project using `composer create-project` to get latest version of eZ Enterprise,
+    you'll need to tell Composer which token to use before the project folder is created in the first place.
+    This can be done in the following way:
+
+    ``` bash
+    COMPOSER_AUTH='{"http-basic":{"updates.ez.no":{"username":"<installation-key>","password":"<token-password>"}}}' composer create-project --keep-vcs --repository=https://updates.ez.no/bul/ ezsystems/ezplatform-ee my-new-ee-project
```

Comment:
We should get rid of `--keep-vcs`, you end up with trial when that is specified as the sources it will pull in from github is always trial, unlike packages.


But getting rid of it means restructuring update doc (see closed PR).

Alternative to this is that we start to ship packages with all licenses and stop having different channels on updates.ez.no for anything but subscription levels (ee vs commerce vs ...).

Reply:
Restructuring the update doc is a larger task, and not likely to be finished soon, while this PR AFAIU temporarily fixes the immediate problem reported in the ticket.